prediction_market_agent/agents/specialized_agent/deploy.py

```diff
@@ -0,0 +1,66 @@
+import typing as t
+from datetime import timedelta
+
+from prediction_market_agent_tooling.deploy.agent import (
+    MAX_AVAILABLE_MARKETS,
+    FilterBy,
+    SortBy,
+)
+from prediction_market_agent_tooling.gtypes import ChecksumAddress
+from prediction_market_agent_tooling.markets.markets import MarketType
+from prediction_market_agent_tooling.markets.omen.omen import OmenAgentMarket
+from prediction_market_agent_tooling.markets.omen.omen_subgraph_handler import (
+    OmenSubgraphHandler,
+)
+from web3 import Web3
+
+from prediction_market_agent.agents.prophet_agent.deploy import (
+    DeployablePredictionProphetGPT4oAgent,
+    DeployablePredictionProphetGPTo1PreviewAgent,
+)
+
+# List of white-listed market creator addresses that these specialized agents will bet on.
+SPECIALIZED_FOR_MARKET_CREATORS: list[ChecksumAddress] = [
+    Web3.to_checksum_address("0xa7E93F5A0e718bDDC654e525ea668c64Fd572882"),
+]
+
+
+class GetSpecializedMarkets:
+    # Do as many bets as we can for the special markets.
+    bet_on_n_markets_per_run = MAX_AVAILABLE_MARKETS
+    # These tends to be long-running markets, it's not interesting to bet on them too much.
+    time_to_bet_again = timedelta(days=7)
```

Comment:
I was thinking for long running markets we might want to define this dynamically based on the duration of the market. e.g. instead have `max_bets_per_market: int = 10`, and then (assuming uniform spacing of bets) we check if a timedelta of `market_duration / 10` has elapsed since last bet. Just a thought for a future change, doesn't need to be part of this PR 😄

Reply:
That makes a lot of sense! If I hadn't merge PMAT PR already, I'd implement it right away, sorry! 😄 

@coderabbitai Please create an issue for Evan's idea!